src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/TseService.java

```diff
@@ -248,6 +252,43 @@ public String formatApplicationResponses(GenericTseApplicationType application,
                 ).collect(Collectors.joining());
     }
 
+    private String getSingleDecisionMarkdown(TseAdminRecordDecisionType decision, String authToken) {
+        List<GenericTypeItem<DocumentType>> documents = decision.getResponseRequiredDoc();
+        List<String[]> documentsRows = addDocumentRows(documents, authToken);
+        List<String[]> rows = new ArrayList<>(List.of(
+            new String[]{"Notification", decision.getEnterNotificationTitle()},
+            new String[]{"Decision", decision.getDecision()},
+            new String[]{"Decision details", decision.getDecisionDetails()},
+            new String[]{"Date", decision.getDate()},
+            new String[]{"Sent by", TRIBUNAL},
+            new String[]{"Type of decision", decision.getTypeOfDecision()}
+        ));
+        rows.addAll(documentsRows);
+        rows.addAll(List.of(
+            new String[]{"Additional information", decision.getAdditionalInformation()},
+            new String[]{"Decision made by", decision.getDecisionMadeBy()},
+            new String[]{"Name", decision.getDecisionMadeByFullName()},
+            new String[]{"Sent to", decision.getSelectPartyNotify()}
+        ));
+
+        return MarkdownHelper.createTwoColumnTable(new String[]{"Decision", ""}, rows);
+    }
+
+    private String formatApplicationDecision(GenericTseApplicationType application, String authToken) {
+
+        if (application.getAdminDecision() == null) {
+            return "";
+        }
+        List<String> decisionsMarkdown = application.getAdminDecision()
+            .stream()
+            .sorted(Comparator.comparing((TseAdminRecordDecisionTypeItem d) -> d.getValue().getDate()).reversed())
+            .limit(2)
+            .map(d -> getSingleDecisionMarkdown(d.getValue(), authToken))
+            .collect(Collectors.toList());
+
+        return String.join("\r\n", decisionsMarkdown);
```

Comment:
Doesn't look like we have a test to make sure this code actually grabs the last two and concats them